azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -625,6 +632,41 @@ public synchronized void prefetchToken(final File tokenFile,
       cred.addToken(jhsdt.getService(), jhsdt);
     }
 
+    if(props.getBoolean(OBTAIN_HIVESERVER2_TOKEN)){
```

Comment:
Should this default to false?

So something like:
```if (props.getBoolean(OBTAIN_HIVESERVER2_TOKEN, false)) {
...
}```

Reply:
Fixed in the new commit.